src/guides/v2.4/graphql/mutations/delete-compare-list.md

```diff
@@ -0,0 +1,72 @@
+---
+group: graphql
+title: deleteCompareList mutation
+contributor_name: Atwix
+contributor_link: https://www.atwix.com/
+---
+
+The `deleteCompareList` mutation deletes the specified comparison list.
+
```

Comment:
Can we emphasize to use this when the guest user signs out. Else there will be unnecessary records, I am not sure if there is a cron in place to clear those.